### PR TITLE
Fix securityContext for KH and few in-built checks for #1025

### PR DESCRIPTION
**Pull request by AshutoshNirkhe**
Manual testing to verify the settings indeed work,

```
$ kubectl get khcheck deployment -n synthetic -o jsonpath='{.spec.podSpec.containers[*].securityContext}'
{"allowPrivilegeEscalation":false,"readOnlyRootFilesystem":true,"runAsNonRoot":true}
```

```
$ kubectl get khcheck dns-status-internal -n synthetic -o jsonpath='{.spec.podSpec.containers[*].securityContext}'
{"allowPrivilegeEscalation":false,"readOnlyRootFilesystem":true,"runAsNonRoot":true}
```

```
$ kubectl get pod kuberhealthy-7f755c7574-fknx6 -n synthetic -o=jsonpath='{.spec.containers[].securityContext}'
{"allowPrivilegeEscalation":false,"readOnlyRootFilesystem":true,"runAsNonRoot":true,"runAsUser":999}
```

```
$ kubectl get pods -n synthetic -o wide
NAME                                     READY   STATUS      RESTARTS   AGE   IP             NODE                             NOMINATED NODE   READINESS GATES
deployment-1653574752                    1/1     Running     0          55s   10.58.71.143   aks-linux1-25436635-vmss000007   <none>           <none>
deployment-deployment-54849d45f6-fj6cn   1/1     Running     0          53s   10.58.71.152   aks-linux1-25436635-vmss000007   <none>           <none>
deployment-deployment-54849d45f6-fmqh4   1/1     Running     0          53s   10.58.71.142   aks-linux1-25436635-vmss000007   <none>           <none>
deployment-deployment-54849d45f6-wjdbd   1/1     Running     0          53s   10.58.71.145   aks-linux1-25436635-vmss000007   <none>           <none>
deployment-deployment-54849d45f6-xdmwd   1/1     Running     0          53s   10.58.71.149   aks-linux1-25436635-vmss000007   <none>           <none>
dns-status-internal-1653574752           0/1     Completed   0          55s   10.58.71.144   aks-linux1-25436635-vmss000007   <none>           <none>
kuberhealthy-656b4845c5-9kwlg            1/1     Running     0          19h   10.58.71.66    aks-xinfra-40004678-vmss00000l   <none>           <none>
kuberhealthy-656b4845c5-m89kb            1/1     Running     0          19h   10.58.71.103   aks-xinfra-40004678-vmss00000m   <none>           <none>
```

### CODE REVIEW

**Comment by AshutoshNirkhe** [comment]
@integrii could you please review ? 
This is partial fix for https://github.com/kuberhealthy/kuberhealthy/issues/1025 we would still need fix on all checks, but this will help our team at the least.

File: deploy/helm/kuberhealthy/Chart.yaml
Comment (integrii) [inline deploy/helm/kuberhealthy/Chart.yaml:3]: Don't rev this version - it will cause the wrong kuberhealthy image to be pulled...

**Review by integrii — CHANGES_REQUESTED** [review]
Looks good other than the `ppVersion` increment!

File: deploy/helm/kuberhealthy/Chart.yaml
Comment (AshutoshNirkhe) [reply deploy/helm/kuberhealthy/Chart.yaml:3]: Guess you mean don't increment/revise this. Let me keep it to v2.7.1 only.

Comment (AshutoshNirkhe) [reply deploy/helm/kuberhealthy/Chart.yaml]: Updated back to v2.7.1

**Comment by AshutoshNirkhe** [comment]
> Looks good other than the `appVersion` increment!

@integrii thanks, have updated that. Can you please check again and merge if all ok!

**Comment by AshutoshNirkhe** [comment]
> > Looks good other than the `appVersion` increment!
> 
> @integrii thanks, have updated that. Can you please check again and merge if all ok!

Hey @integrii , all checks seem to be passing as well now. Could you please review/merge it ?

**Comment by AshutoshNirkhe** [comment]
> > > Looks good other than the `appVersion` increment!
> > 
> > 
> > @integrii thanks, have updated that. Can you please check again and merge if all ok!
> 
> Hey @integrii , all checks seem to be passing as well now. Could you please review/merge it ?

@integrii sorry for bugging again. But are you waiting on something to merge this ? Please let me know if something is needed from my end, thanks.

**Review by integrii — APPROVED** [review]

**Comment by integrii** [comment]
Sorry for being unresponsive.  I started a new Job this week that has kept me occupied.

**Comment by AshutoshNirkhe** [comment]
> Sorry for being unresponsive. I started a new Job this week that has kept me occupied.

No worries, thanks for merging this :) And many congratulations for your new job!

**Pull request merged**
